changelog.md

```diff
@@ -7,6 +7,18 @@ All notable end-user facing changes are documented in this file.
 *Here are all the changes in `master` branch, and will be moved to the appropriate release once they are included in a published nuget package.
 The idea is to track end-user facing changes as they occur.*
 
+### [3.1.6] (changes since 3.1.5)
+
+- Non-breaking improvements
+  - Added eventIndex (#6467)
```

Comment:
We should start editing PR titles so they make more sense without the context of their content